langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaChatModel.java

```diff
@@ -131,18 +139,18 @@ public dev.langchain4j.model.chat.response.ChatResponse chat(final dev.langchain
     @Override
     public Set<Capability> supportedCapabilities() {
         Set<Capability> capabilities = new HashSet<>();
-        if (this.format != null && ResponseFormatType.JSON.equals(this.format.type())) {
+        if (this.responseFormat != null && ResponseFormatType.JSON.equals(this.responseFormat.type())) {
             capabilities.add(RESPONSE_FORMAT_JSON_SCHEMA);
```

Comment:
I guess we should change the logic here and in streaming ollama model (as well as in `OpenAiChatModel` and other models supporting JSON schema, but out of scope of this PR of course) to rely on explicit user configuration:

```java
OllamaChatModel.builder()
    ...
    .supportedCapabilities(Set.of(RESPONSE_FORMAT_JSON_SCHEMA))
    .build();
```

So if user wants AI Service to use/generate JSON schemas for structured outputs, they will need to explicitly enable this capability.

On the other hand, if user wants to specify the response format (e.g., a concrete JSON schema to use), they will specify it like this:
```java
OllamaChatModel.builder()
    ...
    .responseFormat(ResponseFormat.builder().type(JSON).jsonSchema(...).build()) // to use structured outputs
    .responseFormat(ResponseFormat.JSON) // to use json mode
    .format("json") // this should work the same way as .responseFormat(ResponseFormat.JSON) to keep backwards compatibility
    .build();
```

Just in case, response format specified in the `ChatRequest` should always override the one specified in the constructor/builder of the model.

WDYT?

Reply:
Sounds great.
I will change the `supportedCapabilities` setup implementation to the one you suggested.